packages/fiber/src/core/loop.ts

```diff
@@ -14,8 +14,23 @@ let i
 let globalEffects: GlobalRenderCallback[] = []
 let globalAfterEffects: GlobalRenderCallback[] = []
 let globalTailEffects: GlobalRenderCallback[] = []
+
+/**
+ * Adds a global render callback which is called each frame.
+ * @see https://docs.pmnd.rs/react-three-fiber/api/additional-exports#addEffect
+ */
 export const addEffect = (callback: GlobalRenderCallback) => createSubs(callback, globalEffects)
+
+/**
+ * Adds a global after-render callback which is called each frame.
+ * @see https://docs.pmnd.rs/react-three-fiber/api/additional-exports#addAfterEffect
+ */
 export const addAfterEffect = (callback: GlobalRenderCallback) => createSubs(callback, globalAfterEffects)
+
+/**
+ * Adds a global callback which is called when rendering stops.
+ * @see https://docs.pmnd.rs/react-three-fiber/api/additional-exports#addTail
+ */
 export const addTail = (callback: GlobalRenderCallback) => createSubs(callback, globalTailEffects)
 
```

Comment:
I've pre-emptively added section links here as these will need adequate documentation.

I'm wanting to avoid the use of tables without follow-up.